Tests/FuzzilliTests/MutationsTest.swift

```diff
@@ -52,12 +52,17 @@ class MutationsTests: XCTestCase {
         }
         b.endIf()
         let program = b.finalize()
-        program.runtimeTypes = VariableMap<[Int: Type]>([[0: .number], [1: .boolean], [3: .float]])
+        program.runtimeTypes = ProgramTypes(
+            from: VariableMap<Type>([.number, .boolean, .float]),
+            in: program
+        )
 
         let preparedProgram = engine.prepareForMutation(program)
-        XCTAssertEqual(
-            preparedProgram.runtimeTypes, VariableMap<[Int: Type]>([nil, nil, [2: .number], [3: .boolean], [5: .float]])
+        let expectedTypes =  ProgramTypes(
+            from: VariableMap<Type>([nil, nil, .number, .boolean, .float]),
```

Comment:
Nice! Do you think it would be worth adding a VariableMap constructor `VariableMap(_ map: [Int: Value])`, then writing this as
```
from: VariableMap([
    2: .number,
    3: .boolean,
    4: .float,
])
```
Or something like that?